src/fft128/f128_ops.rs

```diff
@@ -705,6 +687,67 @@ pub mod x86 {
         (p, simd.mul_sub_f64x8(a, b, p))
     }
 
+    #[cfg(feature = "nightly")]
+    #[inline(always)]
+    pub(crate) fn quick_two_sum_f64x16(simd: V4, a: f64x16, b: f64x16) -> (f64x16, f64x16) {
+        let s = simd.add_f64x16(a, b);
+        (s, simd.sub_f64x16(b, simd.sub_f64x16(s, a)))
```

Comment:
this looks different than the x8 and x4 variants ? Why is that so ?

Reply:
what's the difference?

Reply:
ah no it's my bad, I misread, all good